src/index.ts

```diff
@@ -1,18 +1,24 @@
 #!/usr/bin/env node
 
-const { execSync } = require("child_process");
-const path = require("path");
-const Discord = require("discord.js");
-const fs = require("fs-extra");
-const prompts = require("prompts");
-const validatePackageName = require("validate-npm-package-name");
+import { execSync } from "child_process";
```

Comment:
can we maintain the old sort order, it seems arbitrary but i like to prioritize node, installed dep, project defined modules, in that order and within each group alphabetized by package name

Reply:
perhaps something i can add to the contributing docs to explain the reason to my madness 😆

Reply:
Shouldn't we also split into groups depending if it is a Typescript import or a `require` import?